src/systemd/isc-dhcp-relay6.service

```
@@ -15,6 +16,6 @@ EnvironmentFile=/run/dhcp-relay/dhcrelay6.conf
 PIDFile=/run/dhcp-relay/dhcrelay6.pid
 ExecStart=/usr/sbin/dhcrelay -6 -pf /run/dhcp-relay/dhcrelay6.pid $OPTIONS
 Restart=always
-
+RestartSec=5
```

```suggestion
RestartSec=10
```
`10s` seems to be the reasonable default most VyOS services use. This will give a bit more time for the PD assignment to go through.